### PR TITLE
Beer and Tell blog post for September 2016.

### PR DESCRIPTION
**Pull request by Osmose**
@emceeaich @groovecoder @bensternthal @craigcook @peterbe @stephaniehobson r?

I'll be posting this on Friday morning, so please leave your feedback before then.

### CODE REVIEW

**Review by groovecoder — APPROVED** [review]

File: beer_and_tell/2016-09-September.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## craigcook: ByeHolly
+Next was [craigcook][], who shared a [virtual yearbook page][] that he made as a tribute to former-Mozillian habber, who recently left. The page shows several photos that are clipped at the edges to look curved like an old television screen. This is done in CSS using [clip-path][] with an [SVG][]-based path for clipping. The SVG used is also defined using proportional units, which allows it to warp and distort correctly for different image sizes, as seen by the variety of images it is used on in the page.
+
+[craigcook]: https://mozillians.org/en-US/u/craigcook/
+[virtual yearbook page]: http://craigcook.github.io/etc/byeholly/
+[clip-path]: https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
+[SVG]: https://developer.mozilla.org/en-US/docs/Web/SVG
+
+## peterbe: react-buggy
+[peterbe][] told us about [react-buggy][], a client for viewing Github issues implemented in [React][]. It is a rewrite of [buggy][], a similar client peterbe wrote for Bugzilla bugs. Issues are persisted in [IndexedDB][] so that the app can function offline. The client also uses [elasticlunr.js][] to provide full-text search on issue titles and comments.
```
Comment (peterbe) [inline beer_and_tell/2016-09-September.md]: s/Github/GitHub

Comment (peterbe) [inline beer_and_tell/2016-09-September.md]: Also, instead of linking to `IndexedDB` can you change that to [Lovefield](https://github.com/google/lovefield) and add in a bracket after "(a wrapper for IndexedDB)".

**Review by peterbe — COMMENTED** [review]
r+ with nits

File: beer_and_tell/2016-09-September.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+[bensternthal]: https://mozillians.org/en-US/u/bensternthal/
+[Nest]: https://nest.com/
+[Weather Underground]: https://www.wunderground.com/
+[InfluxDB]: https://www.influxdata.com/
+[Grafana]: http://grafana.org/
+[Docker]: https://www.docker.com/
+
+## craigcook: ByeHolly
+Next was [craigcook][], who shared a [virtual yearbook page][] that he made as a tribute to former-Mozillian habber, who recently left. The page shows several photos that are clipped at the edges to look curved like an old television screen. This is done in CSS using [clip-path][] with an [SVG][]-based path for clipping. The SVG used is also defined using proportional units, which allows it to warp and distort correctly for different image sizes, as seen by the variety of images it is used on in the page.
```
Comment (craigcook) [inline beer_and_tell/2016-09-September.md]: Once a Mozillian, always a Mozillian! I'd rephrase this as "a farewell tribute to former teammate Holly Habstritt-Gaal, who recently took a job at another company."

File: beer_and_tell/2016-09-September.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+Once a month, web developers from across the Mozilla Project get together to talk about our side projects and drink, an occurrence we like to call "Beer and Tell".
+
+There's a [wiki page available][wiki] with a list of the presenters, as well as links to their presentation materials. There's also a [recording available][recording] courtesy of Air Mozilla.
+
+[wiki]: https://wiki.mozilla.org/Webdev/Beer_And_Tell/2016/September
+[recording]: https://air.mozilla.org/webdev-beer-and-tell-september-2016/
+
+## emceeaich: Gopher Tessel
+First up was [emceeaich][], who shared [Gopher Tessel][], a project for running a [Gopher][] server on a [Tessel][]. Tessel is small circuit board that runs [Node.js][] projects; Gopher Tessel reads sensors (such as the temperature sensor) connected to the board, and exposes their values via Gopher. It also can control lights connected to the board.
```
Comment (emceeaich) [inline beer_and_tell/2016-09-September.md]: There should be some context about Gopher protocol, such as "running a Gopher server, which was an Internet protocol popular before the World Wide Web, and now largely forgotten".

**Review by emceeaich — CHANGES_REQUESTED** [review]
Feedback on Gopher Tessel section
